### PR TITLE
TDD project

### PR DESCRIPTION
**Pull request by Piercel2022**
In this project we implement a class with some methods, but we have done it by doing test-driven development (TDD). We remember to write tests first and then the code. You can check our commits history.

### CODE REVIEW

File: spec/fizzbuzz_spec.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      it 'returns "fizzbuzz" when the number is divisible by 3 and 5' do
+        string = fizzbuzz(15)
+
+        expect(string).to eq('fizzbuzz')
+      end
+      it 'returns "7" in other case' do
+        string = fizzbuzz(4)
+
+        expect(string).to eq('7')
+      end
```
Comment (RolandM99) [inline spec/fizzbuzz_spec.rb]: - [ ] Kindly, make sure that your test for `fizzbuzz` function works as expected. For now, it fails. 
image.

**Review by RolandM99 — CHANGES_REQUESTED** [review]
Hi @Piercel2022, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


### To highlight 💯 
- Good commit message ✔️ 
- No Linter errors ✔️ 
- Follow Github flow ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on @Rolandm99 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by HAMMAS-SALEEM — APPROVED** [review]
# Status: Approved :green_circle: :1st_place_medal: :100:
success

## Hey @Piercel2022, @looh16 :wave:

Great work on implementing the requested changes. Your project is complete. :blush:
There is nothing else to say about your project. :+1:

## Highlights :1st_place_medal:

:heavy_check_mark: Following Requirements
:heavy_check_mark: Following GitFlow
:heavy_check_mark: Descriptive PR Title
:heavy_check_mark: Descriptive PR Summary

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by looh16** [comment]
Thanks!

**Pull request merged**